_locales/zh_CN/messages.json

```diff
@@ -0,0 +1,3358 @@
+  },
+  "usbDeviceOpenFail": {
+    "message": "打开 USB 设备 <span style=\"color: red\">失败</span>"
+  },
+  "usbDeviceClosed": {
+    "message": "关闭 USB 设备<span style=\"color: #37a8db\">成功</span>"
+  },
+  "usbDeviceCloseFail": {
+    "message": "关闭 USB 设备<span style=\"color: red\">失败</span>"
+  },
+  "usbDeviceUdevNotice": {
+    "message": "<strong>udev rules</strong>有没有正确安装？详细信息请查阅文档。"
+  },
+
+  "noConfigurationReceived": {
+    "message": "<span style=\"color: red\">10秒</span>内没有接收到配置信息，飞控通信<span style=\"color: red\">失败</span>"
+  },
+  "firmwareVersionNotSupported": {
+    "message": "<span style=\"color: red\">不支持</span>此固件版本，此地面站仅支持 $1 到 $2 (不包括)"
+  },
+  "firmwareVariantNotSupported": {
+    "message": "<span style=\"color: red\">不支持</span>此固件版本， 请更新到INAV支持固件。 刷新之前请使用CLI命令备份，CLI 备份/还原 命令请参考文档"
+  },
+
+  "tabSwitchConnectionRequired": {
+    "message": "在查看任何页面信息之前，你需要先<strong>连接</strong>"
+  },
+  "tabSwitchWaitForOperation": {
+    "message": "<span style=\"color: red\">不能</span>执行当前操作，请等待上一个操作完成…"
+  },
+
+  "tabSwitchUpgradeRequired": {
+    "message": "需要<strong>更新</strong>到最新版本的 inavFlight 固件才能使用 $1 页面."
+  },
+  "firmwareVersion": {
+    "message": "固件版本：<strong>$1</strong>"
+  },
+  "apiVersionReceived": {
+    "message": "<span style=\"color: #37a8db\">已收到</span>MultiWii API 版本 - <strong>$1</strong>"
+  },
+  "uniqueDeviceIdReceived": {
+    "message": "<span style=\"color: #37a8db\">已收到</span>唯一设备 ID - <strong>0x$1</strong>"
+  },
+  "boardInfoReceived": {
+    "message": "飞控: <strong>$1</strong>, 版本: <strong>$2</strong>"
+  },
+  "buildInfoReceived": {
+    "message": "当前固件发布时间: <strong>$1</strong>"
+  },
+  "fcInfoReceived": {
+    "message": "飞控信息，识别名: <strong>$1</strong>, 版本: <strong>$2</strong>"
+  },
+
+  "notifications_app_just_updated_to_version": {
+    "message": "程序已经更新到版本: $1"
+  },
+  "notifications_click_here_to_start_app": {
+    "message": "单击这里运行程序"
+  },
+
+  "statusbar_port_utilization": {
+    "message": "端口利用率:"
+  },
+  "statusbar_usage_download": {
+    "message": "下行: $1%"
+  },
+  "statusbar_usage_upload": {
+    "message": "上行: $1%"
+  },
+  "statusbar_packet_error": {
+    "message": "数据包错误:"
+  },
+  "statusbar_i2c_error": {
+    "message": "I2C 错误:"
+  },
+  "statusbar_cycle_time": {
+    "message": "循环时间:"
+  },
+  "statusbar_cpu_load": {
+    "message": "CPU 负载: $1%"
+  },
+
+  "dfu_connect_message": {
+    "message": "请使用固件烧写器来访问 DFU 设备"
+  },
+  "dfu_erased_kilobytes": {
+    "message": "擦除 $1 kB 闪存 <span style=\"color: #37a8db\">成功</span>"
+  },
+  "dfu_device_flash_info": {
+    "message": "检测到闪存空间为 $1 kiB"
+  },
+  "dfu_error_image_size": {
+    "message": "<span style=\"color: red; font-weight: bold\">错误</span>: 固件文件大于飞控芯片的闪存。固件大小: $1 kiB, 上限 = $2 kiB"
+  },
+
+  "eeprom_saved_ok": {
+    "message": "EEPROM <span style=\"color: #37a8db\">已保存</span>"
+  },
+
+  "defaultWelcomeIntro": {
+    "message": "欢迎使用<strong>INAV - Configurator</strong>, 为简化固件升级，设置和调校飞控而生的工具。"
```

Comment:
> Again, please convince me you have already thought about risk of tearing user experience. Betaflight configurator using "配置程序" at this string, please offer the reason for changing it back to "configurator".

I suggest you look at what is written in the default language of inav and then discuss this issue. You can’t point something with inav to your betaflight. Again, if you think there is ambiguity, welcome to fork to correct the accusations of maintenance. What can’t be solved?

Reply:
Please do not use the method of transferring topics to avoid my question. Do you take into account the issues I raised above? Please answer my question first, and then I will consider answering your question.

Reply:
I have said it many times, most of them are directly quoted based on betaflight, you see if this is the same
"backupFileIncompatible": {
        "message": "上一版本配置程序生成的备份文件和当前版本的配置程序不兼容"
    },
But some of us think that betaflight is not well understood, then we will replace it with semantics that we think is better understood

Reply:
Great, now please explain this sentence

> I suggest you look at what is written in the default language of inav and then discuss this issue.

Can I understand it as - for Chinese users, "Configurator"is easier to understand than "配置程序"? 

**ARE YOU A NATIVE SPEAKER?**

Except for this one, the software frame we all using will cause an ugly rendering if the words are not in standard UTF-8, especially for Chinese characters. The space I add is NEEDED for almost all strings. I believed you have seen that, but LET ME REMIND YOU ONCE MORE. **MANY SPACEs IN CHINESE TRANSLATION ARE UNDELETABLE.**

Reply:
‘配置器’and ‘配置程序’program, we think it is better to understand the developer ‘配置程序’, but better to understand the white user '配置器', then we have to pay attention to the UTF-8 and space problems you said, and the corresponding will be opened after translation To see the display effect to prevent confusion

Reply:
white user. what the hell are you talking about? If typing in English makes you feel uncomfortable, you can type Chinese here and we won't blame you.

Again, what makes you think you understand the user characteristics and demands of INAV users in Chinese better and change those strings' translation causally?

Reply:
Xiaobai users should be called in Chinese，This translation was also carried out under the guidance of the previous pix and QGC translation team

Reply:
Then why you are playing double standards may I ask?

Reply:
The problem of confusion between ‘配置器’and ‘配置程序’ is not in this string, so I still have to ask you:

Can I understand it as - for Chinese users, "Configurator"is easier to understand than "配置程序"?

ARE YOU A NATIVE SPEAKER?

Don't use the method of transferring topics to avoid my question.

Reply:
Hahaha, unexpectedly, welcome to maintain inav translations in the future, there are still a large number of translations that have not been completed

Reply:
@hjjoe1213123 Maybe “配置工具” is better.

Reply:
then please elaborate and clarify the ambiguity caused by changing this entry and why we using this for years but now you suddenly jumped out and we should change this.

Reply:
1. Not the oldest is the best, if so MWC will be the most popular flight controller today.
2. iNav is very new project for most of chinese users, so it's a good chance to give users a new experience.
3. If most of users think our translation is better, then maybe it's the time you should change yours, otherwise, we will change ours.